arex-schedule-web-api/src/main/java/com/arextest/schedule/service/ReplayCaseRemoteLoadService.java

```diff
@@ -63,7 +63,7 @@ public int queryCaseCount(ReplayActionItem replayActionItem, String providerName
     int queryTotalCount = EMPTY_SIZE;
     try {
       int caseCountLimit =
-          CommonConstant.AUTO_PINED.equals(providerName) ? AUTO_PINED_CASE_LIMIT :
+          CaseProvider.AUTO_PINED.getName().equals(providerName) ? AUTO_PINED_CASE_LIMIT :
```

Comment:
What is the purpose of AUTO_PINED_CASE_LIMIT?

Reply:
auto-pinned cases should ideally ignore case limit, hard-coded 20000 to avoid extreme situation